### DISCUSSION TITLE
"Deprecation" of the /friends endpoint from the Hypixel API

### DISCUSSION THREAD

**Discussion by justinrest** [HypixelDev/PublicAPI#574] [General]
The error message reports that this endpoint has been deprecated, is this true or placeholder? I would be extremely disappointed to see this endpoint removed.

```json
{
    "success": false,
    "cause": "This endpoint has been deprecated and is no longer available."
}
```

Edit: [https://hypixel.net/threads/public-api-removal-of-the-friends-endpoint.5245947/](https://hypixel.net/threads/public-api-removal-of-the-friends-endpoint.5245947/)

**Comment by MattTheCuber** [comment]
Ty developers that abused this!

**Comment by Clemintina** [comment]
The endpoint was deprecated by Hypixel without any notice.

The real value of this endpoint is significant in the Bedwars community. From the Bedwars Winstreaking community's perspective, This change was irrational and stupid. It basically broke winstreaking as it's been the only real solution to detecting snipers. 

From the Hypixel Administration's side, The change was for user privacy however this solution would've been better as an optional toggle like Winstreaks, Online status and Recent Games.
